doc/sphinx/user_manual/basic_starter_tutorial/15_different_segments_in_sections.md

```diff
@@ -0,0 +1,72 @@
+(part:user_manual:chap:basic_starter_tutorial:sec:15_different_segment_in_sections)=
+Different segments in sections
+===============================
+
+Slab and fault may not only vary in depth, but may also vary a lot laterally. To accomplish this in the world builder, we will need to introduce the term sections. Section are a way to overwrite the segments at a specified coordinate. 
+
+```{note}
+One way of thinking about segments and sections is that they for a 2D grid together. The segments form one axis (down dip) and the sections form the other axis (along strike). 
+```
+
+**Sections replace the all segments at that coordinate, and it is required that all sections have the same number of segments**.
+
+In this case we are going to change the values for the first coordinate, which is coordinate 0. We keep the length the same as the default first segment, but we make the second segment a bit shorter. We also do not change the feature composition model in the first segment, but we do change it in the second segment, where we set it equal to composition of the lower part of the oceanic plate for the whole thickness of the segment.
```

Comment:
```suggestion
In this case we are going to change the values for the first coordinate, which is coordinate 0. We keep the length the same as the default first segment (compare lines 48 and 75), but we make the second segment a bit shorter (compare lines 52 and 58). We also do not change the feature composition model in the first segment, but we do change it in the second segment, where we set it equal to composition of the lower part of the oceanic plate for the whole thickness of the segment (line 59).
```

Reply:
I think adding line numbers in here just helps clarify a little bit and will help the reader keep track of what is being kept constant and where the differences in the sections are.

Reply:
Yes, that is great, thanks!